modules/sbo-data-model.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-data-model_{context}"]
+= Data model
+
+Service binding annotations follow the convention:
+
+[source,yaml]
+----
+service.binding(/<NAME>)?: "<VALUE>|(path=<JSONPATH_TEMPLATE>(,objectType=<OBJECT_TYPE)?(,elementType=<ELEMENT_TYPE>)?(,sourceKey=<SOURCE_KEY>)?(,sourceValue=<SOURCE_VALUE>)?)"
```

Comment:
there is no closing `>` for `OBJECT_TYPE`. It should be `<OBJECT_TYPE>`

Reply:
Addressed.